package.json

```diff
@@ -69,6 +70,7 @@
     "@types/redux-form": "^7.2.4",
     "@types/redux-logger": "^3.0.6",
     "@types/redux-saga": "^0.10.5",
+    "@types/reselect": "^2.2.0",
```

Comment:
are these types aligned with `reselect 3.0.1`?

Reply:
The reselect main package already includes types. Going to remove this.